Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/ImmunizationTimelineComponent.vue

```diff
@@ -74,6 +74,22 @@ export default class ImmunizationTimelineComponent extends Vue {
                                     <strong> Lot Number: </strong>
                                     {{ agent.lotNumber }}
                                 </div>
+                                <div
+                                    :data-testid="`immunization-disclaimer-${agent.code}`"
+                                >
+                                    <span>
+                                        <strong
+                                            >You can add or update immunizations
+                                            by visiting
+                                        </strong>
+                                        <a
+                                            href="https://www.immunizationrecord.gov.bc.ca"
+                                            target="_blank"
+                                            rel="noopener"
+                                            >immunizationrecord.gov.bc.ca</a
+                                        >.
```

Comment:
The mockup shows the link with strong styling as well.

Reply:
I'm okay with the way it has been implemented. I'll modify the mock to match implemented!